linera-rpc/src/grpc/conversions.rs

```diff
@@ -1031,11 +1014,11 @@ pub mod tests {
     }
 
     #[test]
-    pub fn validator_name() {
-        let validator_name = ValidatorName::from(ValidatorSecretKey::generate().public());
+    pub fn validator_public_key() {
+        let validator_key = ValidatorPublicKey::from(ValidatorSecretKey::generate().public());
```

Comment:
`let public_key = ValidatorPublicKey`

Reply:
Done in [Fix missed renames](https://github.com/linera-io/linera-protocol/pull/3372/commits/1c4535584e4c5794a6c793a09490c90608de4c9a).